bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -194,8 +194,66 @@ class PartialRowsData(object):
 
     def __init__(self, response_iterator):
         self._response_iterator = response_iterator
+        self.generator = YieldRowsData(response_iterator)
+
         # Fully-processed rows, keyed by `row_key`
         self._rows = {}
+
+    def __eq__(self, other):
+        if not isinstance(other, self.__class__):
+            return False
+        return other._response_iterator == self._response_iterator
+
+    def __ne__(self, other):
+        return not self.__eq__(other)
+
+    @property
+    def state(self):
+        """State machine state.
+
+        :rtype: str
```

Comment:
Here and throughout the rest of the PR, feel free to use Google-style docstrings (vs. the traditional Sphinx-style here). This project is 3.5 years old and we've only switched to **allowing** Google-style docstrings for a year or so. In general, we "require" PRs from Googlers to use Google-style docstrings when adding new methods/functions/classes.